css/main.css

```diff
@@ -357,8 +367,34 @@ table th {
 		padding-bottom:0;
 	}
 	
-	#menu { 
-		display:none; 
+	#menu, 
+	#menu * { 
+		display:none!important;
+		visibility: hidden;
 	}
 	
+	.mod1 { background-color:#000!important; color:#FFFFFF!important; }
```

Comment:
If you give all the modules the class 'mod' or 'module', you can replace all these rules with a single one.